public/index.html

```diff
@@ -26,7 +26,7 @@
     -->
     <title>React App</title>
   </head>
-  <body>
+  <body style="padding-bottom: 50px">
```

Comment:
Please use styling files to make improvements for styling
You shouldn't put inline styling directly to html